ide/provisioning/install-debian-packages.sh

```diff
@@ -2,8 +2,12 @@
 
 set -e
 
+# To prevent apt from displaying dialogs
+DEBIAN_FRONTEND=noninteractive
```

Comment:
Unrelated change, and one without effect.

Reply:
Fixed and moved into a separate commit.